docs/content/en/docs/cluster-operator/kurator-cluster-api.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Deploy Cluster with Kurator Cluster API"
+linkTitle: "Deploy Cluster with Kurator Cluster API"
+description: >
+  The easiest way to deploy cluster with Kurator Cluster API.
+---
+
+In this tutorial we’ll cover the basics of how to use [Cluster API](https://github.com/kurator-dev/kurator/blob/main/pkg/apis/cluster/v1alpha1/cluster_types.go) to provision Kubernetes clusters.
+
+## Prerequisites
+
+{{< boilerplate prerequisites >}}
+
+## Build `cluster-operator` from source
+
+{{< boilerplate build-from-source >}}
+
+## Install cluster operator
```

Comment:
We should have a separate guide for deploying cluster operator, then all the other examples will only need to refer it

Reply:
follow up https://github.com/kurator-dev/kurator/issues/191